docs/features/streaming.rst

```diff
@@ -7,8 +7,73 @@ Overview
 Server Streaming
 ----------------
 
```

Comment:
Add the formal names (unary-unary, unary-stream, stream-unary, stream-stream) of each, so that it may be used elsewhere
add also the GRPCAction property to make it into the matching stream